scripts/db_migrator.py

```diff
@@ -1200,6 +1200,55 @@ def set_version(self, version=None):
         entry = { self.TABLE_FIELD : version }
         self.configDB.set_entry(self.TABLE_NAME, self.TABLE_KEY, entry)
 
+    def enable_per_command_aaa(self):
+        aaa_table = self.configDB.get_table('AAA')
+        if not aaa_table:
+            log.log_info('AAA table does not exist ignore setup per-command accounting&authorization.')
+            return
+
+        authentication_config_old = self.configDB.get_entry('AAA', 'authentication')
+        # If device enabled TACACS authentication, then enable TACACS per-command accounting and authorization
```

Comment:
>enable [](http://example.com/codeflow?start=56&length=6)

The decision should also be driven by GOLDEN_CFG_FILE. GOLDEN_CFG_FILE could express the intention to enable AAA feature or disable, and db_migrator should respect the intention. ref: migrate_telemetry()

Reply:
The old image's enablement does not matter. As long as the required passkey is there, and intention is to enable, then we should enable.

Reply:
Fixed, also discussion with Gang about how to handle version, because AAA table should migrate on all version >= 202205, so add migrate to common_migration_ops.